src/Clock.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component } from 'react';
+
+type State = {
+  today: Date;
+};
+
+type Props = {
+  clockName: string;
+};
+
+const formatTime = (time: Date) => {
+  return time.toUTCString().slice(-12, -4);
+};
+
+export class Clock extends Component<Props, State> {
+  state: Readonly<State> = {
+    today: new Date(),
+  };
+
+  timerCountId = 0;
+
+  componentDidMount() {
+    this.timerCountId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+      window.console.info(formatTime(this.state.today));
```

Comment:
It will print the previous value, not the current one. Create a constant for the value and pass it to setState and console.

https://legacy.reactjs.org/docs/react-component.html#setstate

Reply:
Thank you) Lost it)
Fixed ✔